tests/formulas/formulas.test.ts

```diff
@@ -100,3 +106,29 @@ describe("createAdaptedRanges", () => {
     expect(moveFormula(model, "='Sheet 2'!B2", 1, 10)).toEqual("='Sheet 2'!C12");
   });
 });
+
+describe("Remove columns/rows that are references of formula", () => {
+  let model: Model;
+  beforeEach(() => {
+    model = new Model({
+      sheets: [
+        {
+          colNumber: 10,
+          rowNumber: 10,
+        },
+      ],
+    });
```

Comment:
```suggestion
    model = new Model();
```
we do'nt really care about the number of cols at initialization in those tests